2.7.0/final.md

```diff
@@ -0,0 +1,1073 @@
+- Decoupled python reducer from compilation mode ([#147123](https://github.com/pytorch/pytorch/pull/147123))
+#### FullyShardedDataParallel2 (FSDP2)
+- Clamp `reduce_dtype` in lazy init ([#143297](https://github.com/pytorch/pytorch/pull/143297))
+- Enabled FSDP2 on XPU device ([#143737](https://github.com/pytorch/pytorch/pull/143737))
+- Made post-backward condition more robust ([#144781](https://github.com/pytorch/pytorch/pull/144781))
+- Enabled MTIA device in FSDP2 library code ([#145842](https://github.com/pytorch/pytorch/pull/145842))
+- Avoided resetting version counter of all_gather_output in inference_mode ([#146709](https://github.com/pytorch/pytorch/pull/146709))
+- Supported ignoring parameters in FSDP2 ([#146631](https://github.com/pytorch/pytorch/pull/146631))
+- Enabled FSDP tests on XPU device ([#147518](https://github.com/pytorch/pytorch/pull/147518))
+- Enabled FSDP2 on HPU device ([#148667](https://github.com/pytorch/pytorch/pull/148667))
+#### DTensor
+- Added `aten.amin/amax` to `linear_reduction_strategy` ([#143747](https://github.com/pytorch/pytorch/pull/143747))
+- Added `src_data_rank` to `distribute_tensor` API ([#143883](https://github.com/pytorch/pytorch/pull/143883))
+- Added strategy for `_scaled_mm` ([#143760](https://github.com/pytorch/pytorch/pull/143760))
+- Added `aten.view.dtype` op support ([#144404](https://github.com/pytorch/pytorch/pull/144404))
+- Enabled sharding prop to handle cross mesh computation ([#147869](https://github.com/pytorch/pytorch/pull/147869))
+- Added CuDNN SDPA op support to DTensor ([#148537](https://github.com/pytorch/pytorch/pull/148537))
+- Optimized `shard_dim_alltoall` to use `alltoall_single` ([#148868](https://github.com/pytorch/pytorch/pull/148868))
+- Deprecated `_shard_tensor` to use `src_data_rank=None` ([#144171](https://github.com/pytorch/pytorch/pull/144171))
+- Added pointwise ops strategy for `aten.minimum` ([#145816](https://github.com/pytorch/pytorch/pull/145816))
+#### TensorParallel
+- Propagated `src_data_rank` kwarg in TP API ([#144005](https://github.com/pytorch/pytorch/pull/144005))
+#### Torch Elastic
+- Added kill logic for current process when killing a worker ([#141060](https://github.com/pytorch/pytorch/pull/141060))
+- Made `etcd_rendezvous` publicly importable ([#145396](https://github.com/pytorch/pytorch/pull/145396))
+- Exposed the rendezvous keepalive arguments ([#145228](https://github.com/pytorch/pytorch/pull/145228))
+#### Pipelining
+- Added `generate_stage_to_rank_mapping` utility ([#146193](https://github.com/pytorch/pytorch/pull/146193))
+- Removed `stage_index_to_group_rank` from schedule ([#146217](https://github.com/pytorch/pytorch/pull/146217))
+
+
+## CPU
+#### General
+- Implement blend operation for float, double, int in VEC ATen backend for SVE (#146479)
+#### x86
+- Add support for int8 `brgemm` (#143384)
+- Upgrade submodule oneDNN to v3.7.1 (#148293)
```

Comment:
I think upgrade of oneDNN to 3.7.1 can be moved to be under General heading as it benefits both x86 and AArch64.